### ISSUE TITLE
Megamek hex map is not Accessible to Screen Reader Uesrs

### ISSUE THREAD

**Issue by BlindGuyNW** [MegaMek/megamek#1314]
### Environment
Megamek 0.45.5 Snapshot.
Mac, but applies equally to all.
Java version 1.8.0_161

### Description
Megamek's hex map is currently very difficult to use as a screen reader user, because I am limited to inquiring about hexes one at a time. Some kind of "landmarks," feature, or a way to tell which hexes are actually important would be very useful. This would go hand in hand with the work being done in #1288 to make the Unit display accessible.

See [this video](https://youtu.be/q73QOmEanKo)

**Comment by h-rezaei** [comment]
I'm talking with my friends who also are screen reader users and we're trying to brainstorm  for an idea to make the board readable.
Is it possible to create a different window that shows the board in a text form and allows navigation with either arrow keys (standard) or numbpad keys for better navigation of the board?
